buku.py

```diff
@@ -4002,19 +4001,28 @@ def main():
         argparser.print_help(sys.stdout)
         sys.exit(0)
 
+    # By default, Buku uses ANSI colors. As Windows does not really use them,
+    # we'd better check for known working console emulators first. Currently,
+    # only ConEmu is supported. If the user does not use ConEmu, colors are
+    # disabled unless --colors is specified.
+    if sys.platform == 'win32' and os.environ.get('ConemuDir') is None:
+        if args.colorstr is None:
+            args.nc = True
+
     # Handle color output preference
     if args.nc:
         logging.basicConfig(format='[%(levelname)s] %(message)s')
     else:
         # Set colors
-        ID = setcolors(args.colorstr)[0] + '%d. ' + COLORMAP['x']
+        colorstr = args.colorstr if args.colorstr else 'oKlxm'
```

Comment:
You missed `colorstr_env`. ;)

Reply:
I think we should support `colorstr_env` it in Windows as well. If someone is setting this env var he knows what he is doing.

Reply:
Good catch, thank you. I got lost while trying what would work on both *cmd* and *ConEmu*. Sigh...